src/IO/S3/Requests.h

```diff
@@ -58,6 +58,11 @@ class ExtendedRequest : public BaseRequest
         return BaseRequest::GetChecksumAlgorithmName();
     }
 
+    std::string getRegion() const
```

Comment:
`nit`: since it's returning an override? 
```suggestion
    std::string getRegionOverride() const
```